lib/demos/material/bottom_navigation_demo.dart

```diff
@@ -152,60 +123,48 @@ class _NavigationIconView {
   _NavigationIconView({
     this.title,
     this.icon,
-    TickerProvider vsync,
-  })  : item = BottomNavigationBarItem(
+  }) : item = BottomNavigationBarItem(
           icon: icon,
           title: Text(title),
-        ),
-        controller = AnimationController(
-          duration: kThemeAnimationDuration,
-          vsync: vsync,
-        ) {
-    _animation = controller.drive(CurveTween(
-      curve: const Interval(0.5, 1.0, curve: Curves.fastOutSlowIn),
-    ));
-  }
+        );
 
   final String title;
   final Widget icon;
   final BottomNavigationBarItem item;
-  final AnimationController controller;
-  Animation<double> _animation;
-
-  FadeTransition transition(BuildContext context) {
-    return FadeTransition(
-      opacity: _animation,
-      child: Stack(
-        children: [
-          ExcludeSemantics(
-            child: Center(
-              child: Padding(
-                padding: const EdgeInsets.all(16),
-                child: ClipRRect(
-                  borderRadius: BorderRadius.circular(8),
-                  child: Image.asset(
-                    'assets/demos/bottom_navigation_background.png',
-                    package: 'flutter_gallery_assets',
-                  ),
+
+  Stack destination(BuildContext context) {
```

Comment:
I'm not a big fan of effectively creating a build method inside of a non widget class. Could this be refactored such that this is a `StatelessWidget` that takes in a `BottomNavigationBarItem` in the constructor. Then rename/replace `_navigationViews` with a list of `BottomNavigationBarItem`s.

Reply:
Will do.